communications/newsletters/newsletter_21_Jan2020.md

```diff
@@ -0,0 +1,173 @@
+
+This year, as most of us continue to work from home during the pandemic, we want to:
+
+-   🌍 **Strengthen and provide additional support** for our network of contributors.
+-   ☕️ **Facilitate collaborative efforts** in the community by hosting coworking calls and events.
+-   🏆 Support you in representing _The Turing Way_ to **highlight your work and resources**.
+-   🌸 **Welcome your thoughts and ideas** on how we as a community project can support you.
+-   🗣 **Create more opportunities to listen** to new voices and bring in their perspectives.
+
+We look forward to seeing and welcoming you to our community spaces and events.
+You can reach out to the team by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com), open an [issue on GitHub](https://github.com/alan-turing-institute/the-turing-way/issues), or directly contact Malvika Sharan and Kirstie Whitaker on [Slack](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw). 
+
+
+Read more details on these topics below! 👇
+
+## Community Meetings
+
+![This illustration shows a person working on their computer and talking to multiple people through an online video call, a dog is sitting next to them](images/2021-01-RemoteBookDash1.jpg)
+
+*The Turing Way Community, & Scriberia. (2020, November). Illustrations from The Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.4323154*
+
+### Bimonthly Collaboration Cafes
+The first Collaboration Cafe of this year took place on 20 January 2021. 
+The next one will be on 3 February 2021 from 15:00 - 17:00 UTC ([see in your local time zone](https://arewemeetingyet.com/London/2021-02-03/15:00)). 
+Like last year, we will continue hosting these calls every first and third Wednesday of every month, unless updated differently in [our calendar](https://arewemeetingyet.com/London/2020-12-02/15:00). 
+Find more details on this shared notes: [https://hackmd.io/@turingway/collaboration-cafe](https://hackmd.io/@turingway/collaboration-cafe). 
+
+### Weekly Coworking Calls
+We have also been hosting Coworking Calls every Monday and Thursday from 11:00 - 12:00 UTC. 
+To stay connected with people in Eastern Time Zones, we have added one more session on Wednesdays from 15:00 - 16:00 UTC, except when Collaboration Cafes are planned.
+Find the dates for the upcoming coworking calls on this shared note: [https://hackmd.io/@turingway/coworking-call](https://hackmd.io/@turingway/coworking-call). 
+
+### Join the Spanish Translation Team on Mondays
+Camila Rangel-Smith and Raina Camacho Toro are hosting coworking sessions every Monday at 17:00 UTC to work on the [Spanish translation of the book](https://www.transifex.com/theturingway/theturingway/).
+Connect with them on the Slack channel #traducción-español ([Slack invitation link](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw)) to receive rolling invites to join in their effort.
+
+## News from the community
```

Comment:
Not sure this heading quite fits based on the content below. It feels more like a section on other ways to contribute to the Turing Way - maybe "Other community contribution paths" / "Other community paths" / "More ways to support The Turing Way"?

Reply:
That's a usual header where I post stuff from the community (GitHub and Slack posts). So, maybe for this month, I should change it to fit the details. Thank you for the suggestions.